packages/presentation/lib/utils/formatters.dart

```diff
@@ -0,0 +1,174 @@
+import 'package:flutter/services.dart';
+
+class PhoneInputFormatter implements TextInputFormatter {
+  @override
+  TextEditingValue formatEditUpdate(
+    TextEditingValue oldValue,
+    TextEditingValue newValue,
+  ) {
+    if (newValue.text.length < oldValue.text.length) {
+      return TextEditingValue(
+        text: newValue.text,
+        selection: TextSelection.collapsed(
+          offset: newValue.text.length,
+        ),
+      );
+    }
+
+    final phoneRegExp = RegExp(
```

Comment:
looks top! if it is possible, can u please check with Katya if this regex covers all cases? if it does it - great!